assets/patches/sidecars/provisioner.yaml

```diff
@@ -0,0 +1,46 @@
+spec:
+  template:
+    spec:
+      containers:
+        - name: csi-provisioner
+          image: ${PROVISIONER_IMAGE}
+          imagePullPolicy: IfNotPresent
+          args:
+            - --csi-address=/var/lib/csi/sockets/pluginproxy/csi.sock
+            - --http-endpoint=localhost:${LOCAL_METRICS_PORT}
+            - --leader-election
+            - --leader-election-lease-duration=${LEADER_ELECTION_LEASE_DURATION}
+            - --leader-election-renew-deadline=${LEADER_ELECTION_RENEW_DEADLINE}
+            - --leader-election-retry-period=${LEADER_ELECTION_RETRY_PERIOD}
+            - --leader-election-namespace=openshift-cluster-csi-drivers
+            - --v=${LOG_LEVEL}
+          # Empty env. instead of nil for json patch to append new env. vars there
+          env: []
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /var/lib/csi/sockets/pluginproxy/
+          resources:
+            requests:
+              memory: 50Mi
+              cpu: 10m
+        - name: provisioner-kube-rbac-proxy
```

Comment:
IBM VPC defines only `csi-provisioner` container, not `provisioner-kube-rbac-proxy` one. Should we make `provisioner-kube-rbac-proxy` optional?

Reply:
I would say all CSI drivers should export metrics, i.e. have provisioner-kube-rbac-proxy.

We need to sync with IBM, because they deploy the driver in their own managed OCP that's not HyperShift and I don't know how they collect metrics.